.gitignore

```diff
@@ -1,3 +1,4 @@
+.idea/
```

Comment:
Can you remove this?

Reply:
Why? What's the harm?

Reply:
This is the kind of thing you could exclude in your own personal `~/.gitignore`, FWIW :)

Reply:
I actually wasn't aware of the personal `~/.gitignore`, thanks!

Reply:
Done, thanks for the tip!